design/src/production-chains/raw-resources.md

```diff
@@ -0,0 +1,201 @@
+# Raw Resources
+
+These are the simple, fundamental materials that intermediate and end products are made of.
+Raw resources cannot be derived from another raw resources: they are "elemental" in that way.
+The total amount of each raw resource (other than water and energy) is conserved at each resource processing step.
+
+Different areas of the map will have different quantities of each resource to begin with.
+
+## Core
+
+Resources that should be added in an MVP demo.
+
+### Light
+
+**Acquisition:** It comes from the sun.
+
+**Uses:** Used by plants to capture energy, and provide calories to entire colony.
+
+**Handling properties:** Cannot be tranported. Blocked by shade.
+
+**Storage properties:** Cannot be stored.
+
+**Path to renewability:** The sun really does just keep burning.
+
+**Downsides of excess:** Heat, sun scorch of plants.
+
+### Water
+
+**Acquisition:** Gather from lakes, rivers, soil and rain.
+
+**Uses:** Essential for sustenance of all life. One of the primary transport mechanics of goods.
+
+**Handling properties:** Sinks into soil. Flows downstream. Cannot be carried directly, as it is a fluid.
+
+**Storage properties:** Evaporates over time.
+
+**Path to renewability:** Rainfall, incoming streams.
+
+**Downsides of excess:** Washes away goods. Drowns structures over time.
+
+### Dirt
+
+Planned to split into sand/silt/clay/organic matter after the MVP.
+
+**Acquisition:** Dig up the ground.
+
+**Uses:** Build simple structures. Grow plants and fungi.
+
+**Handling properties:** Solid object.
+
+**Storage properties:** Solid object.
+
+**Path to renewability:** Filter incoming streams.
+
+**Downsides of excess:** Piles up cluttering movement.
+
+### Stone
```

Comment:
I'm not sure how I feel about stone, I think for harder structures, we should prefer calcified or cellulose-heavy organic materials

or thicker plant-materials

Reply:
Stone is an important part of the natural environment, and you see stuff like caddisflies using it directly as a building material. Definitely want organic glues though!

https://en.wikipedia.org/wiki/Structures_built_by_animals#Collected_materials

Reply:
stones could also be worked by fungi to produce things, so i agree they're important!